src/docs/Contributing/Designers/AddNewIcons/index.mdx

```diff
@@ -0,0 +1,61 @@
+---
+title: 'Add new icons'
+order: 4
+---
+
+> First of all, each icon must be reviewed by the designer(s) in charge of the quality and the consistency of the icons at LEROY MERLIN. Second of all, each new icon must be declined in the following sizes: `24px`, `32px`, `48px` & `64px`. Here are the steps that you must follow to add new icons in Mozaic.
+
+We would like to advise you to read this [documentation](/Foundations/Icons/designers/) about icons design principles before getting in touch with Mozaic to add new icons in the design kit. This will definitely help you to understand our vision when it comes to icons and it will make the process way easier for both parties.
+
+## 1. Get in touch with Mozaic's team
+
+It's important to discuss your needs and what can be produced together before starting to create the said icon. Our icon designer is [Simon Roger](mailto:simon.roger@ext.leroymerlin.fr) for now. Get in touch with him to speak about format and sizes of your icon(s). You can also reach the team from Mozaic on different channels. You will more informations about that right [here](https://mozaic.adeo.cloud/Help/).
+
+## 2. Icon design phase
+
+This is the point where the icon is designed either by you or our design team.
+
+## 3. New branch on GitHub
+
+You must create a branch to be able to push your work in Mozaic's design system repository. We advise you to name it `feat-YYWW-icons` (feat-1949-icons) to follow our [branch naming guidelines](/Contributing/Developers/GitConventions/#branch-names).
+
+## 4. Place your icon(s) in Mozaic's repository
+
+Your icon needs to put in the **design-system--styleguide** repository in this folder `designer-system--styleguide/src/icons/`.
+
+## 5. We've built a script for you to search for potential errors
+
+This script will locate errors so you can fix them the easiest way possible. Please type the following lines in your terminal:
+* `cd FolderName/GitHub/design-system--styleguide`
+* `yarn install`
+* `yarn develop`
```

Comment:
the name of the repo is `mozaic-design-system` design-system--styleguide is the old name, GitHub keep track of it, but a new developer would not have this directory. 

not `yarn install` but `yarn lerna` and I think we should just send them to the related page, (follow the developper installation guide) because he need to install lerna, yarn, node, the project etc.. if it is his first time. 

the command for the icons will be : `yarn icons:build`

=> then the script will output errors if there is errors in the icons names or structure like : 

this is the kind of errors you should look for : 

```
✗ ERROR : in MediaFullScreen_32 px.svg : icon Filename should start with a category name separated by _ with the icon name
✗ ERROR : in MediaFullScreen_32 px.svg : icon Filename should not contain spaces
✗ ERROR : in MediaFullScreen_32 px.svg : icon Filename should always end with _[size]px.svg
✗ ERROR : in MediaFullScreen_32 px.svg :  found in /16px/ directory. Icon File name is either wrong or the icon is in the wrong directory
```